src/jdk.management/unix/classes/com/sun/management/internal/OperatingSystemImpl.java

```diff
@@ -93,7 +93,7 @@ public double getContainerCpuLoad() {
                 int totalCPUs = getHostOnlineCpuCount0();
                 int containerCPUs = getAvailableProcessors();
                 // scale the total host load to the actual container cpus
-                hostTicks = hostTicks * containerCPUs / totalCPUs;
+                hostTicks = hostTicks / totalCPUs * containerCPUs;
```

Comment:
It might be better to convert to a floating-point calculation before converting back to a whole number:

`hostTicks = (long) (hostTicks * (1.0 * containerCPUs / totalCPUs));`